olaf/scripts/runner.py

```diff
@@ -0,0 +1,36 @@
+import os
+import spacy
+from abc import ABC, abstractmethod
+from olaf import Pipeline
+from olaf.repository.corpus_loader import TextCorpusLoader
+
+
+class Runner(ABC):
+    def __init__(self, model_name="en_core_web_md"):
+        """Initialise a pipeline Runner.
+
+        Attributes
+        ----------
+        pipeline: Pipeline
+            The pipeline to execute.
+        """
+        spacy_model = spacy.load(model_name)
+
+        corpus_loader = TextCorpusLoader(
+            corpus_path=os.path.join(os.getenv("DATA_PATH"), "demo.txt")
```

Comment:
use the variable `corpus_path` here and precise `corpus_path="demo.txt"` in the init of `demo_pipeline.py`